pkg/sql/logictest/testdata/logic_test/tenant

```diff
@@ -60,6 +60,9 @@ id  name   status
 statement error tenant "seven" does not exist
 SHOW TENANT seven
 
+statement error tenant two does not have replication info
```

Comment:
hmm I would expect the error we hit would indicate there is no active replication job rather than info, wdyt?